tracer.go

```diff
@@ -253,9 +253,16 @@ func (r *tracerImpl) Flush() {
 	}
 
 	if r.reportInFlight == true {
-		maybeLogError(errPreviousReportInFlight, r.opts.Verbose)
 		r.lock.Unlock()
-		return
+		tickerChan := time.Tick(r.opts.MinReportingPeriod / 2)
```

Comment:
Any thoughts on the period here? Also, should we add a max retry and then fail with an error?